github/users_keys.go

```diff
@@ -8,15 +8,17 @@ package github
 import (
 	"context"
 	"fmt"
+	"time"
 )
 
 // Key represents a public SSH key used to authenticate a user or deploy script.
 type Key struct {
-	ID       *int64  `json:"id,omitempty"`
-	Key      *string `json:"key,omitempty"`
-	URL      *string `json:"url,omitempty"`
-	Title    *string `json:"title,omitempty"`
-	ReadOnly *bool   `json:"read_only,omitempty"`
+	ID        *int64     `json:"id,omitempty"`
+	Key       *string    `json:"key,omitempty"`
+	URL       *string    `json:"url,omitempty"`
+	Title     *string    `json:"title,omitempty"`
+	ReadOnly  *bool      `json:"read_only,omitempty"`
+	CreatedAt *time.Time `json:"created_at,omitempty"`
```

Comment:
I believe we have decided to make new time values be of type `Timestamp`.
Could you please change this from `*time.Time` to `*Timestamp`?

Reply:
@gmlewis I've changed type of CreatedAt to *Timestamp